package.json

```diff
@@ -35,6 +35,7 @@
     "react": "0.12.2",
     "serve-static": "1.9.2",
     "shelljs": "0.4.0",
+    "sinon": ">=1.4.0 <2",
```

Comment:
I don't know if this is what you're asking but sinon provides test spies. I believe the tests in this repo, like in current blip, no longer run at all, but @jh-bate would need to confirm. If that's true, I would think we should consider cleaning up by getting rid of outdated tests and dependencies rather than adding a version of something we don't use anymore to the package.json. But I'll let @jh-bate weight in.